__tests__/__snapshots__/flavored-compilers.test.js.snap

```diff
@@ -98,16 +98,18 @@ exports[`ReadMe Magic Blocks Tables 1`] = `
 exports[`ReadMe Magic Blocks Tables with breaks 1`] = `
 "[block:parameters]
 {
+  \\"data\\": {
+    \\"h-0\\": \\"th 1\\",
+    \\"h-1\\": \\"th 2\\",
+    \\"0-0\\": \\"cell 1  \\\\nafter the break\\",
+    \\"0-1\\": \\"cell 2\\"
+  },
+  \\"cols\\": 2,
+  \\"rows\\": 1,
   \\"align\\": [
     \\"center\\",
     \\"center\\"
-  ],
-  \\"cols\\": 2,
-  \\"rows\\": 2,
-  \\"h-0\\": \\"th 1\\",
-  \\"h-1\\": \\"th 2\\",
-  \\"0-0\\": \\"cell 1  \\\\nafter the break\\",
-  \\"0-1\\": \\"cell 2\\"
```

Comment:
There was even an existing snapshot?!

Reply:
🤯

Reply:
Existing, as if I didn't write that test...